mlflow/openai/__init__.py

```diff
@@ -573,25 +573,70 @@ def _load_model(path):
 
 
 def _has_content_and_role(d):
-    return "content" in d and "role" in d
-
-
-class _FormattableMessage:
-    def __init__(self, message):
-        self.content = message.get("content")
-        self.role = message.get("role")
-        self.variables = _parse_format_fields(self.content)
+    return isinstance(d, Dict) and "content" in d and "role" in d
```

Comment:
```suggestion
    return isinstance(d, dict) and "content" in d and "role" in d
```

to avoid confusion. I think `Dict` should be used only for type annotations.